scripts/tests/doppelganger_protection.sh

```diff
@@ -96,7 +96,7 @@ if [[ "$BEHAVIOR" == "success" ]]; then
 
     echo "Starting the last validator client"
 
-    ../local_testnet/validator_client.sh $HOME/.lighthouse/local-testnet/node_4 http://localhost:8100 &
+    ../local_testnet/validator_client.sh $HOME/.lighthouse/local-testnet/node_4 http://localhost:9100 &
```

Comment:
ah looks like there are two more beacon API calls here:

https://github.com/sigp/lighthouse/blob/dd444767c66645f77a10132f97e0d73ae0d77f88/scripts/tests/doppelganger_protection.sh#L113

and 

https://github.com/sigp/lighthouse/blob/dd444767c66645f77a10132f97e0d73ae0d77f88/scripts/tests/doppelganger_protection.sh#L131

Reply:
^ needs to be updated to 9100 too

Reply:
thanks Jimmy, addressed.